public/configs/markets.json

```
@@ -125,6 +132,13 @@
     "whitepaperLink": "https://polkadot.network/PolkaDotPaper.pdf",
     "coinMarketCapsLink": "https://coinmarketcap.com/currencies/polkadot-new/"
   },
+  "DYM-USD": {
+    "name": "Dymension",
+    "tags": [""],
```

```suggestion
    "tags": [],
```

Unless you wanted to assign a different tag